### ISSUE TITLE
Publisher NuGet Package has WordApi.dll/xml and not PublisherApi.dll/xml

### ISSUE THREAD

**Issue by GeoSheehy** [NetOfficeFw/NetOffice#216]
I am trying to use the PublisherAPI NuGet Package but when used I get WordApi. I used the NuGet Package explorer to confirmed this).

![image](https://user-images.githubusercontent.com/5929937/44626007-2238f200-a8e3-11e8-8907-ef8561a51fc7.png)

-George

**Comment by jozefizso** [comment]
Thanks for reporting this bug and I'm sorry for the delay.

I created new package for MS Publisher which contains the correct library.
This package has version **1.7.4.3-hotfix**.

I had to hide the existing _1.7.4.3_ package so Visual Studio does not offer it for installation.

You must enable **Include prerelease** setting in Visual Studio to get this latest version, otherwise you will get 1.7.4.1 only.

Here is the package on NuGet.org: https://www.nuget.org/packages/NetOfficeFw.Publisher/1.7.4.3-hotfix
